test/synth-sm/06_object_sat.c

```diff
@@ -0,0 +1,71 @@
+// RUN: %sea smt %s --step=small -o %t.smt2
+// RUN: %z3 %t.smt2 fp.spacer.order_children=2 2>&1 | OutputCheck %s
+// CHECK: ^sat$
+
+#include "seahorn/seahorn.h"
+
+extern int nd1();
+extern int nd2();
+extern int nd3();
+extern int nd4();
+extern int nd5();
+extern int nd6();
+extern int nd7();
+extern int nd8();
+extern int nd9();
+
+// Object invariant.
+extern int infer(int count, int max);
+int __attribute__((annotate("partial")))
```

Comment:
factor out all attributes into `#define` . It will make it easier to read. Something like
```
#define PARTIAL_FN __attribute__ ...
int PARTIAL_FN inv(int count, int max);
```

Reply:
Good idea. Should this be added to `seahorn.h`? Or just added to the test cases for now?

Reply:
you can add it to seahorn.h for convenience